rpcs3/Emu/RSX/Common/BufferUtils.h

```diff
@@ -2,7 +2,9 @@
 
 #include <vector>
 
-#include "Utilities/GSL.h"
+#define TCB_SPAN_NAMESPACE_NAME gsl
```

Comment:
Is it possible to wrap all this repitition in a header? That way you just include the header as normal and it does the define and include without having to redefine TCB_SPAN_NAMESPACE_NAME in every file that includes the header. Utilities/span.h should be fine.